sorc/ncep_post.fd/SURFCE.f

```diff
@@ -646,7 +648,7 @@ SUBROUTINE SURFCE
         DO J=JSTA,JEND
           DO I=ISTA,IEND
             IF(SMSTAV(I,J) /= SPVAL)THEN
-              GRID1(I,J) = SMSTAV(I,J)*100.
+              GRID1(I,J) = SMSTAV(I,J)
```

Comment:
@EricJames-NOAA This modification will change the results for other non-FV3 models. You might add a condition such as "IF (modelname == "FV3R") " ...